pkg/network/node/egressip.go

```diff
@@ -37,8 +36,9 @@ type namespaceEgress struct {
 type egressIPWatcher struct {
 	sync.Mutex
 
-	localIP string
-	oc      *ovsController
+	localIP   string
+	oc        *ovsController
+	informers common.SDNInformers
 
 	networkClient networkclient.Interface
```

Comment:
`networkClient` is unused now; it's replaced by `informers`. (And you should put `informers` in the group where `networkClient` is now, not the group with `localIP` and `oc`)